robot/Cumulus/tests/browser/relationships/create_relationships.robot

```diff
@@ -48,7 +48,8 @@ Create Relationships for contacts
     ...                                 &{contact2}[FirstName] &{contact2}[LastName]
     ...                                 Parent
 
-    Click Link                          link=Show more actions
+    Click More Actions Button
+    # Click Link                          link=Show more actions
```

Comment:
As mentioned before another comment, that could be removed.

Reply:
done